community/community.hpp

```diff
@@ -78,12 +78,42 @@ class [[eosio::contract("community")]] cambiatus : public eosio::contract
     std::string verification_type; // Can be 'automatic' and 'claimable'
     std::uint8_t is_completed;
     eosio::name creator;
+    std::uint8_t has_proof_photo;
+    std::uint8_t has_proof_code;
+    std::string photo_proof_instructions;
 
     std::uint64_t primary_key() const { return id; }
     std::uint64_t by_objective() const { return objective_id; }
 
     EOSLIB_SERIALIZE(action,
-                     (id)(objective_id)(description)(reward)(verifier_reward)(deadline)(usages)(usages_left)(verifications)(verification_type)(is_completed)(creator));
+                     (id)(objective_id)(description)(reward)(verifier_reward)(deadline)
+                     (usages)(usages_left)(verifications)(verification_type)(is_completed)
+                     (creator)(has_proof_photo)(has_proof_code)(photo_proof_instructions));
+  };
+
+  TABLE newaction
```

Comment:
After the migration this can be removed safely

Reply:
yep, agreed